### ISSUE TITLE
Incorrect date range on the user activity page

### ISSUE THREAD

**Issue by Jain-Aditya** [wikimedia/WikiContrib#98]
> Note: This task is reserved for Google Code-in students. Please do not claim this task. 

Steps to Reproduce: 
* Go to https://tools.wmflabs.org/wikicontrib/
* Provide user information (Fullname, Phabricator and Gerrit usernames)
* You will be redirected to a user activity page that will show "undefined" value for end date. The correct range for the current month should be "January 2019" - "December 2019" (see screenshot below)

![Screen Shot 2019-12-12 at 10 42 26 AM](https://user-images.githubusercontent.com/34097318/70684680-f9519180-1ccc-11ea-994a-ecc063e65e94.png)

Hint: This would likely require minor code changes in the `frontend/WikiContrib-Frontend/src/result.js` file.

**Comment by srish** [comment]
@Jain-Aditya Thank you for reporting the issue! I've imported it here https://codein.withgoogle.com/dashboard/tasks/5550660053893120/preview/. 

@urbanecm @aklapper one of you please help publish this task. thank you!

**Comment by rammanoj** [comment]
This is a result to #102

**Comment by NdibeRaymond** [comment]
Hello, I am an outreachy intern and I would like to work on this issue @srish

**Comment by codeEmpress1** [comment]
Hello, @srish I am an outreachy applicant, can I work on this

**Comment by srish** [comment]
@codeEmpress1 this depends on #102

**Comment by codeEmpress1** [comment]
@srish, Thanks, can I wait for #102 to be resolved, I saw that a discussion is on-going for merging its PR or I choose another issue.

**Comment by srish** [comment]
(closing this as a possible duplicate of 123; can be re-opened if the need arises)
